web/src/lib/tax-access/taxAccess.ts

```diff
@@ -0,0 +1,83 @@
+import { postTaxFilingsOnboarding } from "@/lib/api-client/apiClient";
+import { UpdateQueue } from "@/lib/types/types";
+import {
+  Business,
+  getCurrentBusiness,
+  LookupLegalStructureById,
+  ProfileData,
+} from "@businessnjgovnavigator/shared";
+import { Dispatch, SetStateAction } from "react";
+
+interface TaxAccessProps {
+  profileData: ProfileData;
+  business?: Business;
+  setIsLoading?: Dispatch<SetStateAction<boolean>>;
+  updateQueue?: UpdateQueue;
+}
+
+export const displayBusinessName = (business?: Business): boolean => {
+  return LookupLegalStructureById(business?.profileData.legalStructureId).elementsToDisplay.has(
+    "businessName"
+  );
+};
+
+export const displayResponsibleOwnerName = (business?: Business): boolean => {
+  return LookupLegalStructureById(business?.profileData.legalStructureId).elementsToDisplay.has(
+    "responsibleOwnerName"
+  );
+};
+
+export const taxAccess = async ({
+  business,
+  profileData,
+  setIsLoading,
+  updateQueue,
+}: TaxAccessProps): Promise<void> => {
```

Comment:
We tend to prefer the props variable over this unraveled style. The reason is because it's easier to see if something is passed in. 

(props: TaxAccessProps)

if I see something like props.profileData I know it's passed vs if I see profileData somewhere in this code I might assume it comes from the global context hook